content/docs/connectors/oauth.md

```diff
@@ -48,8 +48,8 @@ connectors:
     # Default: id
     # userIDKey:
 
-    # Auth roviders return non-standard user identity profile
-    # Use claimMapping to map those user infomations to standard claims:
+    # Auth providers return non-standard user identity profile
+    # Use claimMapping to map those user informations to standard claims:
```

Comment:
I'm not a huge fan of `informations` (I prefer using `information` as a mass noun), but using that would require changing `those` to `that` which is more than a simple spelling fix